apps/src/aiComponentLibrary/chatMessage/chat-message.module.scss

```diff
@@ -61,8 +69,24 @@ $assistant-color: $brand_primary_light;
   overflow-x: auto;
 }
 
+%message-container {
+  display: flex;
+  flex-direction: row;
+}
+
+.message-container-assistant {
+  @extend %message-container;
+  justify-content: flex-start;
+}
+
+.messageContainer-user {
```

Comment:
Doesn't this need to be `message-container-user` for the user message style to be applied properly?

Reply:
🤦‍♀️  done